docs/hub/datasets-tags.md

```diff
@@ -0,0 +1,60 @@
+---
```

Comment:
I think we can keep this out of the revamp as dataset search/filters are being revamped soon. WDYT @lhoestq?

Reply:
Yes it's fine to add this later, once the search and filters are updated